lemur/plugins/lemur_aws/plugin.py

```diff
@@ -195,10 +266,26 @@ class AWSSourcePlugin(SourcePlugin):
             "type": "str",
             "helpMessage": "Comma separated list of regions to search in, if no region is specified we look in all regions.",
         },
+        {
+            "name": "path",
+            "type": "str",
+            "helpMessage": "Only discover certificates with this path.",
```

Comment:
would validation add value here?

example usage:
https://github.com/Netflix/lemur/blob/fbf50b365cb0f7a0e9cae31dec1b853b958c45bb/lemur/plugins/lemur_aws/plugin.py#L425

Reply:
Added validation, a default, and consistency with the two plugins.  I also looked at making AWSSourcePlugin choose "/" or "/cloudfront/" default depending on the endpointType, but since we can't really do the same for AWSDestinationPlugin, I feel its better to make users be aware of this explicitly; I added to the helpMessage that this should be /cloudfront/ for CloudFront sources.